WooCommerce/Classes/ViewRelated/Dashboard/MyStore/Cells/ProductTableViewCell.swift

```diff
@@ -42,6 +42,12 @@ class ProductTableViewCell: UITableViewCell {
         }
     }
 
+    var hideBottomBorder: Bool = false {
```

Comment:
nit: the name sounds more like a function, would it be more obvious if it's like `shouldHideBottomBorder` or `hidesBottomBorder`?